Jenkinsfile

```diff
@@ -599,10 +603,16 @@ def withBeatsEnv(Map args = [:], Closure body) {
         if (archive) {
           archiveTestOutput(testResults: testResults, artifacts: artifacts, id: args.id, upload: upload)
         }
-        // Tear down the setup for the permamnent workers.
+        // Tear down the setup for the permanent workers.
         catchError(buildResult: 'SUCCESS', stageResult: 'SUCCESS') {
           fixPermissions("${WORKSPACE}")
-          deleteDir()
+          // TODO: Somehow windows workers got a different opinion regarding removing the workspace
+          // IMPORTANT: windows workers are ephemerals, so this should not really affect us.
```

Comment:
Otherwise, the deleteDir will fail in Windows:

![image](https://user-images.githubusercontent.com/2871786/109680557-84d66d00-7b74-11eb-9539-041a524a7cdd.png)